skimage/transform/_thin_plate_splines.py

```diff
@@ -0,0 +1,120 @@
+# Copyright 2007 Zachary Pincus
+# This file is part of CellTool.
+#
+# CellTool is free software; you can redistribute it and/or modify
+# it under the terms of version 2 of the GNU General Public License as
+# published by the Free Software Foundation.
+
+from scipy import ndimage
+import numpy
+
+def warp_images(from_points, to_points, images, output_region, interpolation_order = 1, approximate_grid=2):
+    """Define a thin-plate-spline warping transform that warps from the from_points
+    to the to_points, and then warp the given images by that transform.
+
+    References
+    ----------
+    .. [1] Bookstein, Fred L. "Principal warps: Thin-plate splines and the
+    decomposition of deformations." IEEE Transactions on pattern analysis and
+    machine intelligence 11.6 (1989): 567-585.
+
+    Parameters
+    ----------
+    from_points: (N, 2) array_like
+            Source image coordinates.
+    to_points: (N, 2) array_like
+            Target image coordinates.
+    image: ndarray
```

Comment:
Currently, this parameter is not reflected in the source code which still expectes a list of `images`.